### PR TITLE
Created critique view for faculty (Can be moved over for admins)

### PR DESCRIPTION
**Pull request by LoganDemaray316**
New view for faculty to create a critique for students that are in their attention box.

Edited files:
* AttentionComponent.vue

Added Files:
* CritiqueFacultyComponent.vue

### CODE REVIEW

File: src/components/CritiqueFacultyComponent.vue [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+                            class="text-darkBlue font-weight-medium pb-2">
+                            {{ songs.at(0).person }}
+                          </v-card-subtitle>
+                        </v-col>
+                      </v-row>
+                    </v-card-text>
+                  </v-card>
+                </v-col>
+              </v-row>
+              <v-row v-if="isExpand">
```
Comment (ethanimooney) [inline src/components/CritiqueFacultyComponent.vue:263]: Is there any reason these can't be rendered conditionally with like a v-for?

You could have a list with all of their titles in it and then render each box based off of that list. Would get rid of the mass of excess code here - this file doesn't need to be 400 lines long.

File: src/components/CritiqueFacultyComponent.vue [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+                <v-col>
+                  <v-card-title class="font-weight-bold text-darkGray pl-1">
+                    <v-row>
+                      <v-col> Deployment </v-col>
```
Comment (ethanimooney) [inline src/components/CritiqueFacultyComponent.vue:267]: Deportment, not deployment

**Review by ethanimooney — CHANGES_REQUESTED** [review]

File: src/components/CritiqueFacultyComponent.vue (hunk 1 above)
Comment (LoganDemaray316) [reply src/components/CritiqueFacultyComponent.vue]: Done

File: src/components/CritiqueFacultyComponent.vue (hunk 2 above)
Comment (LoganDemaray316) [reply src/components/CritiqueFacultyComponent.vue]: Done

**Review by ethanimooney — APPROVED** [review]
LGTM

**Pull request merged**